docs/medical-api/more-info/webhooks.mdx

```diff
@@ -359,3 +359,46 @@ Example payload:
   ]
 }
 ```
+
+#### Bulk Document Download
+
+While you can download specified documents using the [get document url endpoint](/medical-api/api-reference/document/bulk-document-download), 
+patients can have a large number of documents, and it can be tedious to download one at a time. 
+
+Instead trigger a [bulk document download query](/medical-api/api-reference/document/bulk-document-download) - one query per patient at any given time;
+When the result is ready, we’ll send a webhook message with all the signed urls for all the patients documents.
+Example payload:
+
+```json
+{
+"meta": {
+    "messageId": "bd4184dd-3b35-4c60-8130-447d1d528fea",
+    "when": "2023-11-30T05:05:12.215Z",
+    "type": "medical.document-bulk-download"
+  },
+  "patients": [
+    {
+      "status": "completed",
+      "documents": [
+        {
+          "id": "018c1b94-57dc-7950-b564-f74ab9b56cc4",
```

Comment:
we're missing the following fields that are present in the other webhooks for objects in the patient array:

```
      "patientId": "<patient-id-1>",
      "externalId": "<external-id-1>",
```

Reply:
patient id was already there at the bottom but will add external id

Reply:
let's put it at the top for consistency? also, when reading docs, I don't think to scroll to the bottom of an array to view the rest of the properties - will be a better UX to put all of this stuff right at the top going forward